src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -134,6 +135,10 @@ object EasyStageDataset {
       } yield ()
     }
 
+    def isRidirecting(bagRelativePath: String): Try[Boolean] = {
+      Success(s.stageFileDataAsRedirectDatastreams) || (Success(s.stubAVfiles) && isAVType(bagRelativePath))
```

Comment:
you can use the 'hack' described in here: http://stackoverflow.com/a/4982668/2389405; use `(implicit ev: DummyImplicit)` on the `Try[Boolean] 'operator' Boolean` overloads.